action/kafkaProduce.py

```diff
@@ -16,11 +17,19 @@ def main(params):
 
         print "Created producer"
 
+        messageValue = params['value']
+        if 'base64DecodeValue' in params and params['base64DecodeValue'] == True:
+            messageValue = base64.b64decode(params['value'])
```

Comment:
Good one! This can fail and, as written when you made this comment, the invocation would have failed with potentially cryptic message. I have enhanced this situation to fail in a more informative way:

```json
    "response": {
        "status": "application error",
        "statusCode": 0,
        "success": false,
        "result": {
            "error": "value parameter is not base64 encoded"
        }
    }
```